serving/docs/lmi_new/user_guides/lmi-dist_user_guide.md

```diff
@@ -2,8 +2,79 @@
+
+
+### Quantization
+
+Currently, we allow customer to use `option.quantize=<quantization-type>` or `OPTION_QUANTIZE=<quantization-type>` to load a quantized model in `lmi-dist`.
+
+We support the following `<quantization-type>`
+* bitsandbytes (LMI container versions >= 0.24.0)
+* bitsandbytes8 (LMI container versions >= 0.25.0)
+* gptq (LMI container versions >= 0.24.0)
+* awq (LMI container versions >= 0.26.0)
+
+When using pre-quantized models make sure to use the correct model artifacts e.g. `TheBloke/Llama-2-13B-chat-GPTQ`, `TheBloke/Llama-2-13B-chat-AWQ`.
+
+
 ## Model Artifact Structure
+`lmi-dist` expects the model to be [standard HuggingFace format](../deployment_guide/model-artifacts.md#huggingface-transformers-pretrained-format).
 
 ## Quick Start Configurations
 
-## Advanced Configurations
\ No newline at end of file
+You can leverage `lmi-dist` with LMI using the following starter configurations:
+
+### serving.properties
+
+```
+engine=MPI
+option.tensor_parallel_degree=max
+option.rolling_batch=lmi-dist
+option.model_id=<your model id>
+option.max_rolling_batch_prefill_tokens=4096
+```
+
+### environment variables
+
+```
+SERVING_LOAD_MODELS=test::MPI=/opt/ml/model
+OPTION_TENSOR_PARALLEL_DEGREE=max
+OPTION_ROLLING_BATCH=lmi-dist
+OPTION_MODEL_ID=<your model id>
+OPTION_MAX_ROLLING_BATCH_PREFILL_TOKENS=4096
+```
+
+You can use the [SageMaker deployment template](../README.md#using-the-sagemaker-python-sdk-to-deploy-your-first-model-with-lmi) to deploy the model with environment variables.
+
+
+| Item                                                              | LMI Version | Required   | Description                                                                                                                                                                                                                                                                                                                                                                                           | Example value          |
+|-------------------------------------------------------------------|-------------|------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------|
+| option.quantize                                                   | \>= 0.23.0  | No         | Quantize the model with the supported quantization methods                                                                                                                                                                   | Default: `None`   |
```

Comment:
@siddvenk if we are recommending to use env var instead of serving.properties, we should take an action item to  update these tables everywhere in the docs to use `OPTION_*` format. I stumbled across when writin trt-llm docs as well.